src/index.html

```diff
@@ -17,16 +20,25 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <form data-qa="big">
+      <label class="search-bar page__search">
+        <input
+          class="search-bar__input"
+          type="text"
+          data-qa="keypress"
+          placeholder="Try “Los Angeles“"
+        />
+      </label>
+    </form>
 
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+    <form data-qa="small">
+      <label class="search-bar search-bar--size--small page__search">
+        <input
+          class="search-bar__input search-bar__input--size--small"
```

Comment:
```suggestion
          class="search-bar__input search-bar__input--small"
```
Classnames can be simplified